cli/azd/test/functional/env_test.go

```diff
@@ -230,16 +274,58 @@ func envSetValue(ctx context.Context, t *testing.T, cli *azdcli.CLI, key string,
 	require.NoError(t, err)
 }
 
-func envGetValues(ctx context.Context, t *testing.T, cli *azdcli.CLI, args ...string) map[string]string {
-	defaultArgs := []string{"env", "get-values", "--output", "json"}
+func envGetValues(
+	ctx context.Context,
+	t *testing.T,
+	cli *azdcli.CLI,
+	args ...string) map[string]string {
+	defaultArgs := []string{"env", "get-values"}
 	args = append(defaultArgs, args...)
 
 	result, err := cli.RunCommand(ctx, args...)
 	require.NoError(t, err)
 
-	var envValues map[string]string
-	err = json.Unmarshal([]byte(result.Stdout), &envValues)
-	require.NoError(t, err)
+	outputMode := azdcli.GetOutputFlagValue(args)
+
+	envValues := map[string]string{}
+
+	switch outputMode {
+	case "json":
+		err = json.Unmarshal([]byte(result.Stdout), &envValues)
+		require.NoError(t, err)
+	case "", "none":
+		scanner := bufio.NewScanner(strings.NewReader(result.Stdout))
+		for scanner.Scan() {
+			line := scanner.Text()
+			if !strings.Contains(line, "=") {
+				t.Fatal("unexpected line in output: " + line)
+			}
+
+			parts := strings.Split(line, "=")
+			require.Len(t, parts, 2)
```

Comment:
I guess this could fail in cases where you have an equals sign in the value of an envar (which I would guess may be somewhat common, since you run into that in say base64 padding cases) but also this is just test code so I am guessing we will just not construct tests in such a way.

Reply:
I think I might end up changing this to a closer expected format of `.env`.

Reply:
This may be fine for now. `godotenv` actually allows inputs that have multiple `=` characters, or variable names that start with digits ([gist](https://gist.github.com/weikanglim/678b68876d1a62d9e5a306e29ab32001)). Perhaps we need to tighten our validation of `.env` in product code?

Reply:
Let's revisit this if needed in the near future. The test covers against spurious outputs. The format of `env` is largely consistent with `godotenv`